build-tools/create-dotnet-msi/create-dotnet-msi.csproj

```diff
@@ -9,6 +9,13 @@
   <Import Project="..\..\Configuration.props" />
   <Import Project="..\scripts\XAVersionInfo.targets" />
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.VisualStudioEng.MicroBuild.Core" Version="0.4.1">
```

Comment:
Since the `0.4.1` version is in a few places, should we put a `$(MicroBuildVersion)` property in `Configuration.props`?